tests/test_cache.py

```diff
@@ -114,66 +125,21 @@ def test_update_point_cache_signal(mock_datetime):
 
     assert django_cache.get(key) == "Some time"
 
-# @patch("wazimap_ng.cache.update_profile_cache")
-# def test_profile_indicator_updated(mock_update_profile_cache):
-#     instance = namedtuple("profile_indicator", "profile")
-#     instance.profile = "Some profile"
-
-#     cache.profile_indicator_updated("sender", instance)
-
-#     assert mock_update_profile_cache.assert_called_with("Some profile")
-
-# @patch("wazimap_ng.cache.update_profile_cache")
-# def test_profile_highlight_updated(mock_update_profile_cache):
-#     instance = namedtuple("profile_highlight", "profile")
-#     instance.profile = "Some profile"
-
-#     cache.profile_highlight_updated("sender", instance)
-
-#     assert mock_update_profile_cache.assert_called_with("Some profile")
-
-# @patch("wazimap_ng.cache.update_profile_cache")
-# def test_profile_category_updated(mock_update_profile_cache):
-#     instance = namedtuple("profile_category", "profile")
-#     instance.profile = "Some profile"
-
-#     cache.profile_category_updated("sender", instance)
-
-#     assert mock_update_profile_cache.assert_called_with("Some profile")
-
-# @patch("wazimap_ng.cache.update_profile_cache")
-# def test_profile_subcategory_updated(mock_update_profile_cache):
-#     subcategory = namedtuple("profile_subcategory", "category")
-#     category = namedtuple("profile_category", "profile")
-#     subcategory.category = category
-#     category.profile = "Some profile"
-
-#     cache.profile_subcategory_updated("sender", subcategory)
-
-#     assert mock_update_profile_cache.assert_called_with("Some profile")
-
-# @patch("wazimap_ng.cache.update_profile_cache")
-# def test_profile_keymetrics_updated(mock_update_profile_cache):
-#     keymetric = namedtuple("key_metric", "profile")
-#     keymetric.profile = "Some profile"
-
-#     cache.profile_keymetrics_updated("sender", keymetric)
-
-#     assert mock_update_profile_cache.assert_called_with("Some profile")
-
-# @patch("wazimap_ng.cache.update_point_cache")
-# def test_point_updated_location(mock_update_point_cache):
-#     location = namedtuple("location", "category")
-#     location.category = "Some category"
-
-#     cache.point_updated_location("sender", location)
-
-#     assert mock_update_point_cache.assert_called_with("Some category")
-
-# @patch("wazimap_ng.cache.update_point_cache")
-# def test_point_updated_category(mock_update_point_cache):
-#     category = "Some category"
-
-#     cache.point_updated_category("sender", category)
 
-#     assert mock_update_point_cache.assert_called_with("Some category")
+@pytest.mark.django_db
+class TestCache(unittest.TestCase):
+    @patch('wazimap_ng.cache.update_profile_cache', autospec=True)
+    def test_indicator_updated(self, mock_update_profile_cache):
```

Comment:
the test name is not very descriptive. what do you think about: `test_invalidate_profile_cache_for_indicator`

and can you add a test where you add more factories with a different indicator to make sure those don't get invalidated?